sample/ProgressBarSample/MainPage.xaml

```diff
@@ -12,89 +12,124 @@
-
-    <!-- Simple usage -->
-    <VerticalStackLayout Spacing="20">
-
-      <Label Text="Simple usage:" FontSize="Large" />
-
-      <maui:ProgressBar
-        Margin="15,0"
-        HeightRequest="5"
-        Progress="{Binding Path=Value, Source={x:Reference ProgressSlider}}"
-        UseRange="False"
-        RoundCaps="True"
-        UseGradient="{Binding Path=IsChecked, Source={x:Reference GradientCheckBox}}"
-        ProgressColor="{StaticResource ProgressColor}"
-        GradientColor="{StaticResource GradientColor}" />
-
-      <Slider
-        x:Name="ProgressSlider"
-        Minimum="0"
-        Maximum="1"
-        Value="0.25" />
+  <ScrollView>
+    <VerticalStackLayout
+      Padding="20,0"
+      VerticalOptions="Center"
+      Spacing="50">
+
+      <HorizontalStackLayout Spacing="6">
+        <Label VerticalOptions="Center" Text="Use Gradient?" />
+        <CheckBox x:Name="GradientCheckBox" VerticalOptions="Center" />
+      </HorizontalStackLayout>
+
+      <!-- Simple usage -->
+      <VerticalStackLayout Spacing="20">
+
+        <Label Text="Simple usage:" FontSize="Large" />
+
+        <maui:ProgressBar
+          Margin="15,0"
+          HeightRequest="5"
+          Progress="{Binding Path=Value, Source={x:Reference ProgressSlider}}"
+          UseRange="False"
+          RoundCaps="True"
+          UseGradient="{Binding Path=IsChecked, Source={x:Reference GradientCheckBox}}"
+          ProgressColor="{StaticResource ProgressColor}"
+          GradientColor="{StaticResource GradientColor}" />
+
+        <Slider
+          x:Name="ProgressSlider"
+          Minimum="0"
+          Maximum="1"
+          Value="0.25" />
+
+      </VerticalStackLayout>
+
+      <!-- Ranged usage -->
+      <VerticalStackLayout Spacing="20">
+
+        <Label Text="Ranged usage:" FontSize="Large" />
+
+        <maui:ProgressBar
+          x:Name="RangeProgressBar"
+          Margin="15,0"
+          HeightRequest="5"
+          UseRange="True"
+          RoundCaps="True"
+          LowerRangeValue="{Binding Path=Value, Source={x:Reference LowerRangeSlider}}"
+          UpperRangeValue="{Binding Path=Value, Source={x:Reference UpperRangeSlider}}"
+          UseGradient="{Binding Path=IsChecked, Source={x:Reference GradientCheckBox}}"
+          ProgressColor="{StaticResource ProgressColor}"
+          GradientColor="{StaticResource GradientColor}" />
+
+        <Label Text="Lower Range" />
+        <Slider
+          x:Name="LowerRangeSlider"
+          Minimum="0"
+          Maximum="1"
+          Value="0.4" />
+
+        <Label Text="Upper Range" />
+        <Slider
+          x:Name="UpperRangeSlider"
+          Minimum="0"
+          Maximum="1"
+          Value="0.75" />
+
+      </VerticalStackLayout>
+
+      <!--  Segment usage  -->
+      <VerticalStackLayout Spacing="20">
+        <Label Text="Segment usage:" FontSize="Large" />
+
+        <maui:ProgressBar x:Name="SegmentProgressBar"
+                          HeightRequest="5"
```

Comment:
please use the same indentation style as the XAML already has _(if there are more than two attributes, break after the opening tag)_.